mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/massindexing/spi/PojoMassIndexer.java

```diff
@@ -79,6 +79,8 @@ public interface PojoMassIndexer {
 	 * @param dropAndCreateSchema if {@code true} the indexes and their schema will be dropped then re-created before starting the indexing
 	 * @return {@code this} for method chaining
 	 */
+	// TODO: Yoann, feels like we are talking about ORM-triggered indexing here, but we are in a pojo base, drop the example,
+	//  or replace it with a longer one like "(other thread closed its {@code SearchSession} and is processing a {@code SearchIndexingPlan}, ...)"
```

Comment:
Are you talking about this line:

> 	 * so this should only be used when failures of concurrent operations on the indexes (automatic indexing, ...)

If so, actually the current phrasing is too restrictive: it should be talking about all indexing based on indexing-plan, be it explicit or through listeners.

Anyway... I'd just use something less specific:

> 	 * so this should only be used when failures of concurrent operations on the indexes (indexing caused by entity changes, ...)